src/scripts/SpellHandlers/PaladinSpells.cpp

```diff
@@ -381,6 +392,8 @@ void SetupPaladinSpells(ScriptMgr* mgr)
     mgr->register_script_effect(20271, &JudgementLightWisdomJustice); //light
     mgr->register_script_effect(53408, &JudgementLightWisdomJustice);
     mgr->register_script_effect(53407, &JudgementLightWisdomJustice);
+	
+	mgr->register_script_effect(63521, &GuardedByTheLight);
```

Comment:
We use 4 spaces instead of 1 tab.
Check out your msvc configuration:
Go to Tools->Options->Text Editor->All Languages
![tabs](http://i.imgur.com/uVrOdqS.png)

Tools->Options->Text Editor->C/C++->Tabs should look similar to this.